lib/src/widgets/components/request_item_card/request_item_card.dart

```diff
@@ -51,6 +52,8 @@ class RequestItemCard extends StatelessWidget {
                     RequestItemCardDescription(item: item),
                     const SizedBox(height: RequestSizes.s8),
                     RequestItemCardCommentCount(item: item),
+                    const SizedBox(height: RequestSizes.s8),
+                    RequestItemCardOwner(userName: item.creator.username),
```

Comment:
It would be nice to have the owner and the comment count next to each other. Where the owner is on the left, and the comment count is on the right.